widgets/managers/src/ExecuteWorkflowIcon.tsx

```diff
@@ -1,15 +1,13 @@
-import type { ComponentProps } from 'react';
+import type { WorkflowsMenuProps } from '../../common/src/executeWorkflow/WorkflowsMenu';
 
 const WorkflowsMenu = Stage.Common.Workflows.Menu;
 
 export default function ExecuteWorkflowIcon({
     onClick = _.noop,
     workflows = []
 }: {
-    onClick?: (workflow: Parameters<ComponentProps<typeof WorkflowsMenu>['onClick']>[0]) => void;
-    workflows: ComponentProps<typeof WorkflowsMenu>['workflows'];
+    onClick?: WorkflowsMenuProps['onClick'];
+    workflows: WorkflowsMenuProps['workflows'];
 }) {
-    return !_.isEmpty(workflows) ? (
-        <WorkflowsMenu workflows={workflows} onClick={workflow => onClick(workflow)} />
-    ) : null;
+    return !_.isEmpty(workflows) ? <WorkflowsMenu workflows={workflows} onClick={onClick} /> : null;
 }
```

Comment:
What do you think about explicit imports of the lodash functions?
(as suggested in our [Refactors Guidelines](https://cloudifysource.atlassian.net/wiki/spaces/RD/pages/2148073473/Ongoing+Refactors+in+Cloudify+UI))

```suggestion
import { isEmpty, noop } from 'lodash';
import type { WorkflowsMenuProps } from '../../common/src/executeWorkflow/WorkflowsMenu';

const WorkflowsMenu = Stage.Common.Workflows.Menu;

export default function ExecuteWorkflowIcon({
    onClick = noop,
    workflows = []
}: {
    onClick?: WorkflowsMenuProps['onClick'];
    workflows: WorkflowsMenuProps['workflows'];
}) {
    return !isEmpty(workflows) ? <WorkflowsMenu workflows={workflows} onClick={onClick} /> : null;
}
```

Reply:
Make sense, changed to explicit imports.